code/modules/surgery/organs/dwarf_organ.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/organ/alcoholvessel //essentially the opposite of the xeno's plasmavessel, but with alcohol
```

Comment:
During the test merge, I had one implanted in me. Sadly, it does not display the BoozeUI. Is there a way to tie that to the organ?